src/Symfony/Bundle/FrameworkBundle/Tests/Controller/ControllerTest.php

```diff
@@ -37,10 +41,99 @@ public function testForward()
         $container->expects($this->at(0))->method('get')->will($this->returnValue($requestStack));
         $container->expects($this->at(1))->method('get')->will($this->returnValue($kernel));
 
-        $controller = new Controller();
+        $controller = new TestController();
```

Comment:
Instead of creating a class, where we will add all methods progressively, what about using reflection to access the method?

```php
$m = new \ReflectionMethod('...Controller', 'getUser');
$m->setAccessible(true);
$m->invoke(...);
```